app/components/Form/reportingRequirementFormChangeFunctions.ts

```diff
@@ -113,24 +120,37 @@ export const stageReportFormChanges = async (
   const completedPromises: Promise<void>[] = [];
 
   formChangeEdges.forEach(({ node }) => {
+    const defaultVariables = {
+      input: {
+        id: node.id,
+        formChangePatch: {
+          changeStatus: "staged",
+        },
+      },
+    };
     if (node.changeStatus === "pending") {
       const promise = new Promise<void>((resolve, reject) => {
-        mutationFn({
-          variables: {
-            reportType: reportType,
-            input: {
-              id: node.id,
-              formChangePatch: {
-                changeStatus: "staged",
-              },
+        if (reportType)
```

Comment:
Why do we need this condition here. The logic isn't obvious, it could use a comment